pocs/scheduler/__init__.py

```diff
@@ -22,8 +23,8 @@ def create_scheduler_from_config(config, observer=None):
         return None
 
     if not observer:
-        logger.info("No valid Observer found.")
-        return None
+        site_details = create_location_from_config(config)
```

Comment:
```suggestion
        logger.debug(f'No Observer provided, creating from config.')
        site_details = create_location_from_config(config)
```

Reply:
We can have plenty of debug level comments, especially if doing something the user might not expect.